package.json

```diff
@@ -8,20 +8,21 @@
   "main": "dist/index.js",
   "module": "dist/esm/index.js",
   "sideEffects": false,
-  "types": "dist/ts/index.d.ts",
+  "types": "dist/esm/ts/index.d.ts",
   "typesVersions": {
     ">=4.1": {
       "dist": [
-        "dist/ts-4.1/index.d.ts"
+        "dist/esm/ts-4.1/index.d.ts"
       ],
-      "dist/ts/*": [
-        "dist/ts-4.1/*"
+      "dist/esm/ts/*": [
+        "dist/esm/ts-4.1/*"
```

Comment:
Yup, before types were generated as part of the CJS build, now they're part of the ESM build. So they have to be in a subfolder of the ESM build for "rollup reasons":tm:.
And we can't move them back up because then the sourcemaps would not point to the correct relative paths any more.

Reply:
Thanks for the clarification!